js/app.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+
+var itemListArray = [];
+
+//////////////////////////////////////////
+////////////// Constructor
+//////////////////////////////////////////
+
+function Item(itemName, itemPath){
+  this.itemName       = itemName;
+  this.itemPath       = itemPath;
```

Comment:
back the indentations to their associated properties to make it easier to associate the property to value.